pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -94,6 +97,7 @@ func NewReconciler(
 	nodeName types.NodeName,
 	desiredStateOfWorld cache.DesiredStateOfWorld,
 	actualStateOfWorld cache.ActualStateOfWorld,
+	populatorHasAddedPods func() bool,
```

Comment:
I don't think we need lock before reading/writing that variable. afaict - it is not being called concurrently by any other goroutine. 

The volume manager calls `dswp.Run` and that in turn loops the function infinitely between the sleep durations. I don't see any concurrent execution of `dswp.Run` ..